packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -371,18 +378,28 @@ function SingleForm({ form, appUrl, Page }: SingleFormComponentProps) {
                       {t("test_preview")}
                     </Button>
                   </div>
+                  {!form.routes?.some(
+                    (r) => ("isFallback" in r && !r.isFallback) || !("isFallback" in r)
```

Comment:
There is a isFallbackRoute method. That takes care of this TypeScript requirement.
<img width="714" alt="Screenshot 2023-05-11 at 12 47 14 PM" src="https://github.com/calcom/cal.com/assets/1780212/53601c8e-a3cc-4cf7-8c66-10edc04abba3">